### DISCUSSION TITLE
Good web mailing interface

### DISCUSSION THREAD

**Discussion by bagheriali2001** [postalserver/postal#2664] [Help with using Postal]
Hi everyone.

Yesterday, I found out about Postal in my monthly review of [awesome-selfhosted](https://github.com/awesome-selfhosted/awesome-selfhosted), in the [Communication - Email - Complete Solutions](https://github.com/awesome-selfhosted/awesome-selfhosted#communication---email---complete-solutions) section. I had some problems getting it up and running, but I finally made it work. Then I realized that Postal doesn't have any web mailing interface. I am very fond of it and want to use Postal, but I can't find any web mailing interface. Do you guys have any suggestions?
(I have a docker-only rule, all of the services in my server are dockerized)

**Comment by willpower232** [comment]
Postal isn't really designed for daily email sending, it mostly enables software (and printers etc) to send emails without worrying about things such as inboxes or folders of emails.

You could certainly manipulate Postal to do that but if you're looking for webmail interfaces, you would essentially be using something that sends emails with something else that sends emails so it would probably get confusing.
